lab_5_scrapper/target_score.txt

```diff
@@ -1 +1 @@
-6
\ No newline at end of file
+10
```

Comment:
are you planning to increase the mark? now you have got 8 according to the spreadsheet

Reply:
nono i just forgot to change it